temp/credentials.json

```diff
@@ -0,0 +1 @@
+{"installed":{"client_id":"173231060398-b9qcjp82ek4bp23nke32eectad5dpsh6.apps.googleusercontent.com","project_id":"torrentaccessmethod","auth_uri":"https://accounts.google.com/o/oauth2/auth","token_uri":"https://oauth2.googleapis.com/token","auth_provider_x509_cert_url":"https://www.googleapis.com/oauth2/v1/certs","client_secret":"P_hChwDC_akoMJdNsS10prY3","redirect_uris":["urn:ietf:wg:oauth:2.0:oob","http://localhost"]}}
```

Comment:
Was thinking about this, and we can password-protect the client secret, so that we have a way of limiting access to torrent uploads. We haven't seen abuse yet, but there's no sense in inviting it. We can do that in a later effort.

Look into the pyca cryptography library's "recipe" for password-based encryption here: https://cryptography.io/en/latest/fernet/#using-passwords-with-fernet

Basically, we would keep the encrypted client key in this repo and share the password more selectively.

Reply:
Anyway, for now let's not check credentials.json in, revoke the access in Drive, make a new client secrets file and we'll use that. Might as well do it right.

Sorry for not addressing this sooner